WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/BaseStatsUseCaseTest.kt

```diff
@@ -55,19 +55,19 @@ class BaseStatsUseCaseTest : BaseUnitTest() {
         block.fetch(false, false)
         advanceUntilIdle()
 
-        assertData(0, localData)
+        assertData(0, remoteData)
     }
 
     @Test
-    fun `on fetch returns null item when DB is empty`() = test {
+    fun `on fetch uses remote data when DB is empty`() = test {
```

Comment:
np: Does `...returns remote data` instead of `uses remote data` sound better to you?